app/util/citybikes.js

```diff
@@ -168,3 +172,10 @@ export const hasStationCode = vehicleRentalStation => {
     !isNaN(parseFloat(vehicleRentalStation.stationId))
   );
 };
+
+/** *
+ * Parse id number from stationId.
+ */
+export const parseVehicleIdNumber = stationId => {
```

Comment:
There is actually a feedScopedIdUtils.js file with `getIdWithoutFeed` function that we could use instead of this. There is a small upside in having a specific util for this purpose as then it's more clear what we want to get out of the id but still I think maybe we should just reuse the existing util.